### DISCUSSION TITLE
Need help to build the app pls!

### DISCUSSION THREAD

**Discussion by mounaloop** [creepymonster/GlucoseDirect#594] [Q&A]
Hi guys,
I really need some help to build the app so I can use it with my iAPS urgently please. I tried to follow the same steps as I did to build both Loop and iAPS but I get stuck halfway. I forked the main branch and added my six secrets under Secrets and varialbles but when I go to Actions to validate and build, I don't get the workflow menu I'm used to. I appreciate your help.
Thanks!

**Comment by crebitio** [comment]
I was hoping I could just download and compile in Xcode but…
`No profiles for 'com.NB9G889RYZ.libre-direct' were found
Xcode couldn't find any iOS App Development provisioning profiles matching 'com.NB9G889RYZ.libre-direct'.`
and 
`Cannot create a iOS App Development provisioning profile for "com.NB9G889RYZ.libre-direct".
Personal development teams, do not support the Time Sensitive Notifications and Near Field Communication Tag Reading capabilities.`

So I would need a non-personal developer account and generate a provisioning profile? Does this sound right?
